packages/application-shell/src/test-utils/test-utils.spec.js

```diff
@@ -264,3 +274,117 @@ describe('router', () => {
     expect(history.location.pathname).toBe('/foo');
   });
 });
+
+describe('custom render function', () => {
+  describe('wrapper', () => {
+    describe('when provided', () => {
+      const Context = React.createContext();
+      const ProvidedWrapper = ({ children }) => (
+        <Context.Provider value="provided wrapper">{children}</Context.Provider>
+      );
+      ProvidedWrapper.propTypes = {
+        children: PropTypes.node.isRequired,
+      };
+
+      it('should renderApp merges the provided wrapper with its own wrapper', () => {
+        const TestComponent = () => {
+          // provided wrapper
+          const value = useContext(Context);
+          // own wrapper
+          useIntl();
+
+          return value;
+        };
+
+        const rendered = renderApp(<TestComponent number={1} />, {
+          wrapper: ProvidedWrapper,
+        });
+        rendered.getByText(/provided wrapper/i);
+      });
+
+      it('should renderAppWithRedux merges the provided wrapper with its own wrapper', () => {
+        const TestComponent = () => {
+          // provided wrapper
+          const value = useContext(Context);
+          // own wrapper
+          useSelector(() => {});
+
+          return value;
+        };
+
+        const rendered = renderAppWithRedux(<TestComponent number={1} />, {
+          wrapper: ProvidedWrapper,
+        });
+        rendered.getByText(/provided wrapper/i);
+      });
+    });
+
+    describe('when not provided', () => {
+      it.each([renderApp, renderAppWithRedux])(
+        'should %p still manage',
+        renderFn => {
+          const TestComponent = ({ number }) => {
+            return number;
+          };
+
+          const rendered = renderFn(<TestComponent number={1} />);
+          rendered.getByText(/1/);
+        }
+      );
+    });
+  });
+
+  describe('rerender', () => {
+    it('should works with renderApp', () => {
+      const TestComponent = ({ number }) => {
+        // the error won't be triggered unless one of the providers is used
+        useIntl();
+        return number;
+      };
+
+      const rendered = renderApp(<TestComponent number={1} />);
+      rendered.getByText(/1/);
+
+      rendered.rerender(<TestComponent number={2} />);
+      rendered.getByText(/2/);
+      expect(rendered.queryByText(/1/)).not.toBeInTheDocument();
+    });
+
+    it('should works with renderAppWithRedux', () => {
```

Comment:
Grammar seems off here.

Reply:
The whole test will have as title 

*rerender should work with renderAppWithRedux*

I will remove the `s` in `works` which was a typo but other than that I think the grammar is correct. I'm open to any suggestions.